web-app/js/portal/cart/WpsDownloadHandler.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.cart');
+
+Portal.cart.WpsDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
```

Comment:
Don't bother too much extracting common code with that and the gogoduck handler - gogoduck handler should die sooner than later.